pytradfri/kelvin.py

```diff
@@ -0,0 +1,28 @@
+from .const import (ATTR_LIGHT_COLOR_X as X, ATTR_LIGHT_COLOR_Y as Y)
+
+
+KNOWN_XY = {
+  2200: {X: 33135, Y: 27211},
+  2700: {X: 30140, Y: 26909},
+  4000: {X: 24930, Y: 24694},
+}
+
+
+def can_dekelvinize(kelvins):
+    return min(KNOWN_XY.keys()) <= kelvins <= max(KNOWN_XY.keys())
+
+
+def dekelvinize(kelvins):
```

Comment:
Can we rename this? It's not so much a `dekelvinize` as a `kelvin_to_xy` function.

Which means that we should also rename this file to `color.py`, to make it more reusable in the future.